src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -252,4 +252,10 @@
   <data name="PX1032Title" xml:space="preserve">
     <value>DAC properties cannot contain method invocations</value>
   </data>
+  <data name="PX1030Fix" xml:space="preserve">
+    <value>Replace with the PXUnboundDefaultAttribute or set PersistingCheck to Nothing</value>
+  </data>
+  <data name="PX1030Title" xml:space="preserve">
+    <value>Incorrect PXDefaultAttribute to exisiting field </value>
```

Comment:
@k-s-popova , please check this string.

Reply:
@saloponov, I would change the message as follows:
The PXDefault attribute of the field is used incorrectly